main.go

```diff
@@ -116,7 +116,7 @@ func run(ctx context.Context, wg *sync.WaitGroup, globalCollectionOpts state.Col
 			}
 			return
 		} else if globalCollectionOpts.TestRunLogs {
-			runner.TestLogsForAllServers(servers, globalCollectionOpts, logger)
+			reloadOkay = doLogTest(servers, globalCollectionOpts, logger)
 			return
 		} else {
 			var allFullSuccessful bool
```

Comment:
GitHub won't let me comment on the lines below, but should we consider the `doLogTest` return value for the regular test? Right now we have `reloadOkay = allFullSuccessful && allActivitySuccessful` on line 134--should we add the log test result to the conjunction?

Reply:
We intentionally don't fail for the regular test command, since you may not have Log Insights enabled on your plan (which will make the log test fail when its getting the log grant). In those situations we still want `--test` to be successful (i.e. issue a reload), but `--test-logs` should fail (and not reload).

Thus, the code as written here is correct I think. We could add a comment explaining this for future reference in the `--test` command call of `doLogTest` (where we ignore the return value).